tutorials/2. Advanced PyDEns.ipynb

```diff
@@ -0,0 +1,800 @@
+{
```

Comment:
<p>The last sentence: in fact, parameters <strong>are used </strong>as inputs of the neural approximator. Otherwise, we wouldn't be able to solve the equation for different values of parameters.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/pydens/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>